102-CMW-Star-Cloud-Monitoring/index.html

```diff
@@ -0,0 +1,138 @@
+            border: 1px solid #ddd;
+            border-radius: 8px;
+            background-color: #f4f4f4;
+        }
+
+        .section img {
+            width: 100%;
+            height: auto;
+            max-width: 500px;
+            margin: 0 auto;
+            display: block;
+            border-radius: 8px;
+        }
+
+        .section h2 {
+            margin-bottom: 10px;
+            text-align: center;
+            color: #333;
+        }
+
+        .section p {
+            margin-bottom: 10px;
+            text-align: justify;
+        }
+
+        @media (min-width: 768px) {
+            .section {
+                display: flex;
+                align-items: center;
+            }
+
+            .section img {
+                width: 50%;
+            }
+
+            .section div {
+                width: 50%;
+                padding-left: 20px;
+            }
+        }
+    </style>
+</head>
+<body>
+    <header>
+        <h1>Welcome to CMW - Cloud Monitoring Website</h1>
+    </header>
+
+    <div class="container">
+        <div class="section" id="section1">
+            <img src="https://via.placeholder.com/500" alt="Cloud Monitoring">
+            <div>
+                <h2>Real-time Monitoring</h2>
+                <hr/>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. 
+                    Praesent libero. Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet. 
+                    Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper porta. Mauris massa.
+                </p>
+            </div>
+        </div>
+
+        <div class="section" id="section2">
+            <img src="https://via.placeholder.com/500" alt="Data Analytics">
+            <div>
+                <h2>Advanced Data Analytics</h2>
+                <hr/>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. 
+                    Praesent libero. Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet. 
+                    Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper porta. Mauris massa.
+                </p>
+            </div>
+        </div>
+
+        <div class="section" id="section3">
+            <img src="https://via.placeholder.com/500" alt="Alerts and Notifications">
+            <div>
+                <h2>Alerts and Notifications</h2>
+                <hr/>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. 
+                    Praesent libero. Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet. 
+                    Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper porta. Mauris massa.
+                </p>
+            </div>
+        </div>
+    </div>
+
+    <script>
```

Comment:
Similar to CSS, please extract your JS to a separate file and in HTML just import it.